cocos/editor-support/cocosbuilder/CCScale9SpriteLoader.cpp

```diff
@@ -25,7 +25,7 @@ ui::Scale9Sprite* Scale9SpriteLoader::createNode(cocos2d::Node* /*pParent*/, coc
 
 void Scale9SpriteLoader::onHandlePropTypeSpriteFrame(Node * pNode, Node * pParent, const char * pPropertyName, SpriteFrame * pSpriteFrame, CCBReader * ccbReader) {
     if(strcmp(pPropertyName, PROPERTY_SPRITEFRAME) == 0) {
-        ((cocos2d::ui::Scale9Sprite *)pNode)->setSpriteFrame(pSpriteFrame);
+        static_cast<cocos2d::ui::Scale9Sprite*>(pNode)->setSpriteFrame(pSpriteFrame, Rect::ZERO);
```

Comment:
Is Scale9Sprite::setSpriteFrame(spriteFrame) removed? Why should add a Rect::ZERO parameter here?